src/main/java/com/uber/jenkins/phabricator/PhabricatorBuildWrapper.java

```diff
@@ -114,6 +119,10 @@ public Environment setUp(AbstractBuild build,
             if (!createCommit) {
                 patchCommand.add("--nocommit");
             }
+            // Use a globally-configured conduit token, if applicable
+            if (!CommonUtils.isBlank(conduitToken)) {
+                patchCommand.add("--conduit-token=" + this.getConduitToken());
```

Comment:
Yea, I left that as a comment on the overall PR. We probably want to filter this out, although in general I think it's useful to have the commands being run being displayed. @joshuaspence?